drivers/timer/Kconfig

```diff
@@ -165,6 +165,19 @@ config NRF_RTC_TIMER
 	  Counter NRF_RTC1 and provides the standard "system clock driver"
 	  interfaces.
 
+config STM32_LPTIM_TIMER
+	bool "STM32 Low Power Timer"
+	depends on SOC_FAMILY_STM32
```

Comment:
Lets limit to SOC_SERIES_STM32L4X for now

Reply:
CONFIG_STM32_LPTIM_TIMER is defined in the application (sample/subsys/power/stm32_pm/) to avoid disturbing other examples with lptimer